lrelease.pri

```diff
@@ -0,0 +1,36 @@
+# Copyright 2005-2017 The Mumble Developers. All rights reserved.
+# Use of this source code is governed by a BSD-style license
+# that can be found in the LICENSE file at the root of the
+# Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+# This .pri files finds a suitable protoc binary on the system.
+#
+# It exposes the found protoc binary as the 'PROTOC' qmake variable.
+# This variable contains the absolute path to a working protoc binary.
+#
+# Why do we need to do this?
+#
+# The protobuf compiler's binary name in MinGW packages for Linux
+# cross-compiling is never 'protoc', but something like
+# 'x86_64-w64-mingw32.static-protoc'.
+#
+# This file tries to use the right binary, by searching in $PATH
+# using the 'which' tool.
+#
+# If no binary is found, an error is shown to the user.
+#
+# This script also supports setting a MUMBLE_PROTOC environment
+# variable to specify an absolute path to a protoc binary to use.
+# Setting this environment variable overrides any querying.
+
+include(qt.pri)
+
+isEmpty(QMAKE_LRELEASE) {
+	isEqual(QT_MAJOR_VERSION, 5) {
```

Comment:
I think this file can simply be:

    isEmpty(QMAKE_LRELEASE) {
        QMAKE_LRELEASE = $$[QT_INSTALL_BINS]/lrelease
    }

for all Qt versions. Qt 4 has QT_INSTALL_BINS too.